imports/plugins/core/checkout/client/components/completedOrder.js

```diff
@@ -34,213 +34,212 @@ class CompletedOrder extends React.Component {
+                           {shipment.address.city},
+                           {shipment.address.region}
+                           {shipment.address.postal}
+                           {shipment.address.country}
+                         </p>
+                       </div>
+                     </div>);
+                 }
+               })
+               }
+             </div>
 
-            <div className="payment-info" >
-              <div className="order-details-content-title" >
-                <p>
-                  <Components.Translation
-                    defaultValue="Payment Method"
-                    i18nKey="cartCompleted.paymentMethod"
-                  />
-                </p >
-              </div>
-              {
-                this.state.paymentMethods.map(paymentMethod => (
-                  <CompletedOrderPaymentMethod
-                    key={paymentMethod.key}
-                    paymentMethod={paymentMethod}
-                  />))
-              }
-            </div>
-          </div>
-          <CompletedOrderSummary
-            shops={
-              this.state.shops
-            }
-            orderSummary={
-              this.state.orderSummary
-            }
-            isProfilePage={
-              this.state.isProfilePage
-            }
-          />
+             <div className="payment-info" >
+               <div className="order-details-content-title" >
+                 <p>
+                   <Components.Translation
+                     defaultValue="Payment Method"
+                     i18nKey="cartCompleted.paymentMethod"
+                   />
+                 </p >
+               </div>
+               {
+                 this.state.paymentMethods.map(paymentMethod => (
+                   <CompletedOrderPaymentMethod
+                     key={paymentMethod.key}
+                     paymentMethod={paymentMethod}
+                   />))
+               }
+             </div>
+           </div>
+           <CompletedOrderSummary
+             shops={
+               this.state.shops
+             }
+             orderSummary={
+               this.state.orderSummary
+             }
+             isProfilePage={
+               this.state.isProfilePage
+             }
+           />
 
 
-          { /* This is the right side / side content */}
-          <br />
-          <div className="row">
-            <div className="col-md-12">
-              {
+           { /* This is the right side / side content */}
+           <br />
+           <div className="row">
+             <div className="col-md-12">
+               {
 
-                this.state.isCancelled ?
-                  " " :
-                  (<CancelOrderButton isCancelled={this.state.isCancelled} cancelOrder={this.cancelOrder} order={this.state.order} />)
-              }
-            </div>
-          </div>
-        </div>
-      </div>
-    );
-  }
+                 this.state.isCancelled &&
```

Comment:
I believe this should be `!this.state.isCancelled` as the button should not be shown when `isCancelled` is true. Is this right?